src/components/HomePage.jsx

```diff
@@ -223,256 +297,253 @@ class HomePage extends Component {
+								<Link href="https://kubernetes.io" className={classes.link} target="_blank">
+									Kubernetes cluster
+								</Link>{" "}
+								with
+								<Link href="https://www.docker.com" className={classes.link} target="_blank">
+									{" "}
+									Docker
+								</Link>{" "}
+								container technology.
 							</Typography>
-							<Box style={{margin:"1em auto"}}>
-								<Button href={config.slackInvitationLink}
+						</div>
+						{/*Version Block*/}
+						<div>
+							<div>
+								<Typography
+									variant="h5"
+									style={{ cursor: "pointer", textAlign: "center" }}
+									onClick={this.toggleCollapse}
+								>
+									What&apos;s new in <span className="greenText">IN-CORE 4.8?</span>
+									<IconButton>
+										{this.state.open ? <ArrowDropUpIcon /> : <ArrowDropDownIcon />}
+									</IconButton>
+								</Typography>
+								<Typography variant="subtitle1" style={{ textAlign: "center" }}>
+									Current Version: {this.state.githubVersions["in-core"]}
+								</Typography>
+							</div>
+							<Collapse in={this.state.open}>
+								<div className={classes.versionSection}>
+									{this.state.repos.map((repo) => (
+										<div className={classes.versionLine}>
+											<Typography variant="body1" className={classes.versioning}>
+												{repo.title}
+												{/*if version exists, display version; otherwise not displaying the chip*/}
+												{this.state.githubVersions &&
+												this.state.githubVersions[repo.repoName] ? (
+														<Chip
+															size="small"
+															color="primary"
+															label={this.state.githubVersions[repo.repoName]}
+															className={classes.versioning}
+														/>
+												) : null}
+
+												{Object.keys(repo.options).map((option) => (
+													<Link
+														color="primary"
+														underline="always"
+														className={classes.versioning}
+														href={repo.options[option]}
+														target="_blank"
+													>
+														{option}
+													</Link>
+												))}
+											</Typography>
+										</div>
+									))}
+								</div>
+							</Collapse>
+						</div>
+						{/* Getting Started Block */}
+						<div className={classes.infoBlock}>
+							<div>
+								<h2>Getting Started</h2>
+								<p>Here are two simple steps to quickly get you up and running with IN-CORE:</p>
+								<ol>
+									<li>
+										<Link href={config.signUpURL} target="_blank" className={classes.link}>
+											Sign up
+										</Link>
+										&nbsp;for an IN-CORE account
+									</li>
+									<li>
+										Install&nbsp;
+										<Link href={config.pyIncoreDocUrl} target="_blank" className={classes.link}>
+											pyincore
+										</Link>
+										&nbsp;pyincore, a python package that contains service classes to connect with
```

Comment:
"pyincore" is repeated here
<img width="290" alt="image" src="https://github.com/IN-CORE/incore-ui/assets/13950475/6717337f-6e33-47e5-8598-5e37d7db3303">